huggingface/pytorch/buildspec.yml

```diff
@@ -29,3 +44,36 @@ images:
       *CUDA_VERSION, '-', *OS_VERSION ]
     docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, 
       *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+  BuildHuggingFacePytorchCPUPTInferencePy3DockerImage:
+    <<: *INFERENCE_REPOSITORY
+    build: &HUGGINGFACE_PYTORCH_CPU_INFERENCE_PY3 false
+    image_size_baseline: 4899
+    device_type: &DEVICE_TYPE cpu
+    python_version: &DOCKER_PYTHON_VERSION py3
+    tag_python_version: &TAG_PYTHON_VERSION py36
+    os_version: &OS_VERSION ubuntu18.04
+    transformers_version: &TRANSFORMERS_VERSION 4.6.0
+    inference_toolkit_version: &INFERENCE_TOOLKIT_VERSION 1.0.0
```

Comment:
Not sure if we should this in the `dockerfile` or in the `buildpsec.yaml`

Reply:
Good point. Question of usability of Dockerfiles vs code manageability as this version number is used in all the containers. Would recommend leaving it here for now and moving it into Dockerfiles if need arises.